### PR TITLE
blog(embertimes191): uELS 3.0.5 vscode

### PR DESCRIPTION
**Pull request by chrisrng**
## Ember Times review template

- [x] Add your name to top and bottom
- [x] Put emoji in writeup title
- [x] Add a short blurb (could be the title) to the beginning
- [x] Link to external article/repo/etc in paragraph/body text, not just the writeup title (link in writeup title is optional now)
- [x] Add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account
- [x] Check that all links work

### CODE REVIEW

File: content/the-ember-times-issue-191.md [hunk 1]
```diff
@@ -11,6 +12,7 @@ tags:
 👋 Emberistas! 🐹
 
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+Release of uELS 3.0.5 published to to VSCode ✅
```
Comment (ijlee2) [inline content/the-ember-times-issue-191.md]:
```suggestion
Release of uELS 3.0.5 published to VSCode ✅
```

File: content/the-ember-times-issue-191.md [hunk 2]
```diff
@@ -26,15 +28,13 @@ tags:
 
 ---
 
-## [🐹 2. Section title in sentence case](section-url)
+## [Release of uELS 3.0.5 published to to VSCode ✅](https://discord.com/channels/480462759797063690/480499624663056390/896699510888226826)
```
Comment (ijlee2) [inline content/the-ember-times-issue-191.md]:
```suggestion
## [Release of uELS 3.0.5 published to VSCode ✅](https://discord.com/channels/480462759797063690/480499624663056390/896699510888226826)
```

File: content/the-ember-times-issue-191.md [hunk 3]
```diff
@@ -26,15 +28,13 @@ tags:
...
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Unstable Ember Language Server](https://marketplace.visualstudio.com/items?itemName=lifeart.vscode-ember-unstable) is a fully featured fork of Ember Language Server. While the name says _unstable_ it actually is **stable** and was recently [published v3.0.5 to VSCode](https://marketplace.visualstudio.com/items?itemName=lifeart.vscode-ember-unstable)!
```
Comment (ijlee2) [inline content/the-ember-times-issue-191.md]:
```suggestion
[Unstable Ember Language Server](https://marketplace.visualstudio.com/items?itemName=lifeart.vscode-ember-unstable) is a fully featured fork of Ember Language Server. While the name says _unstable_, it actually is **stable** and recently [published v3.0.5 to VSCode](https://marketplace.visualstudio.com/items?itemName=lifeart.vscode-ember-unstable)!
```

File: content/the-ember-times-issue-191.md [hunk 4]
```diff
@@ -26,15 +28,13 @@ tags:
...
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+Thank you to [Alex Kanunnikov (@lifeart)](https://github.com/lifeart) for the updates such as: async file system access, improved startup performance, fixed found issues, and multiple file system providers support.
+
+As part of the release, it added experimental [VSCode web-version support](https://code.visualstudio.com/api/extension-guides/web-extensions) which allows uELS to be integrated into any browser environment.
```
Comment (ijlee2) [inline content/the-ember-times-issue-191.md]: I think we can either add a comma before the word `which`, or start a new sentence to make the paragraph easy to read.

```suggestion
As part of the release, it added experimental [VSCode web-version support](https://code.visualstudio.com/api/extension-guides/web-extensions). This allows uELS to be integrated into any browser environment.
```

**Review by ijlee2 — APPROVED** [review]
Looks good, thanks for the write-up!

**Comment by jaredgalanis** [comment]
@chrisrng it looks like 3.0.6 was just announced as well. Did you have time to pick that up as well? I think we're planning to publish a post this Friday.

**Pull request merged**